demo/App.tsx

```diff
@@ -6,10 +6,77 @@
  *
  */
 
+import * as Constants from './src/constants';
+import Hyperview from 'hyperview';
 import './src/gesture-handler';
+import React, { useState } from 'react';
+import { fetchWrapper, formatDate } from './src/helpers';
+import {
+  SafeAreaInsetsContext,
+  SafeAreaProvider,
+} from 'react-native-safe-area-context';
+import HyperviewScreen from './src/HyperviewScreen';
+import { View } from 'react-native';
 import Navigator from './src/Navigator';
-import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
 
-export default function App() {
-  return <Navigator />;
+const NAMESPACE_URI = 'https://instawork.com/hyperview-set-navigator-source';
```

Comment:
let's use a namespace that's more generic and that isn't coupled to Instawork
```suggestion
const NAMESPACE_URI = 'https://hyperview.org/demo';
```